### ISSUE TITLE
Unlinkified links and missing links in Further Reading sections

### ISSUE THREAD

**Issue by mstimberg** [NeuromatchAcademy/course-content#844]
Hi everyone. According to the contribution guidelines, you prefer issues to pull requests (even though this might only be aimed at the *content* of the lessons), so here we go:
Most of the tutorials have a `further_reading.md` file with a reference list, and most of them have links to the material (DOI etc.). Unfortunately, these are a bit inconvient to use, since the links are not "linkified", i.e. you have to copy & paste them. Many markdown renderers (including GitHub's) automatically linkify URLS, but [MyST syntax](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#markdown-links-and-referencing) requires them to be written as `<https://example.com>` or `[link text](https://example.com)`.
I think it would be helpful to do this (running a little script) for all the references in the further reading sections.
My suggestion would be to let general links (e.g. to a book on an author's website) keep the full URL, but use e.g. `[doi:10.1038/s41583-018-0038-8](https://doi.org/10.1038/s41583-018-0038-8)` ([doi:10.1038/s41583-018-0038-8](https://doi.org/10.1038/s41583-018-0038-8)) for DOI.
I also thought of something else: many of the references are open access, but some of them aren't, and it might be helpful for students  which is which. How about putting a corresponding logo (the PLoS "open padlock" or "closed padlock" logos) at the end of the link? The information should be available via the [unpaywall](https://unpaywall.org/) API. In fact, for the non-open access references, this API can sometimes also give an open access alternative, e.g. a preprint. I can imagine that many students would benefit from this, since their university might otherwise not provide the access they need.

Sorry about the long post, I hope I am not bothering you with something you consider unimportant. If you are interested in these changes, would you prefer a big pull request with everything I mentioned above (again, assuming that you think it is useful)? Or would you rather prefer several pull requests?

**Comment by spirosChv** [comment]
Hey, @mstimberg thank you very much for your comments/suggestions. Indeed all these points are valid and important. I have in my to-do list to update the links, but if you want you can proceed!

Just a personal taste: I prefer to have the links as:

```
doi: [10.1113/jphysiol.1952.sp004764](https://doi.org/10.1113/jphysiol.1952.sp004764)
```

A big pull request with links and logos would be great, then you can ping me to approve it.

**Comment by mstimberg** [comment]
Ah, I started working on this, but I just saw that you already fixed most of the issues :) Are you still interested in the preprint/logo thing, or do you prefer working on this as well?

**Comment by mstimberg** [comment]
Closed via #853 and #850.
